packages/@aws-cdk/lambda-layer-awscli/package.json

```diff
@@ -83,27 +83,33 @@
   },
   "dependencies": {
     "@aws-cdk/aws-lambda": "0.0.0",
+    "@aws-cdk/asset-awscli-v1": "0.0.0",
```

Comment:
This needs to be updated once https://github.com/cdklabs/awscdk-asset-awscli/pull/43/files is released

Reply:
I tested this PR by `yarn link`ing to my local version of @aws-cdk/asset-awscli-v1